core.md

```diff
@@ -371,23 +416,44 @@ placed into a weakmap.
 
 ## Prototypes
 
-FIXME
+All typed objects have an accompanying `prototype`. The `[[Prototype]]` of new instances of a type is set to that `prototype`. For struct types, the `prototype`'s own `[[Prototype]]` is immutably set to `StructType.prototype`. The `[[Prototype]]` of arrays of a struct type `Point`, instantiated using `new Point.array()`, is set to `Point.array.prototype`. That object's own `[[Prototype]]` is set to `StructType.array.prototype`.
```

Comment:
I changed this to `type definitions`, as that's used elsewhere and, I think, makes the most sense.